tests/unit/routes/test_link_router.py

```diff
@@ -421,22 +537,32 @@ def test_link_enhanced_algorithm(
             "/link/fhir", json={"bundle": bundle_4, "algorithm": "dibbs-enhanced"}
         )
         person_4 = response_4.json()["person_reference_id"]
-        assert response_4.json()["is_match"]
+        assert response_4.json()["patient_reference_id"] and uuid.UUID(response_4.json()["patient_reference_id"])
         assert person_4 == person_1
+        assert response_4.json()["prediction"] == "match"
+        assert len(response_4.json()["results"]) == 1
 
         bundle_5 = test_bundle
         bundle_5["entry"] = [entry_list[4]]
         response_5 = client.post(
             "/link/fhir", json={"bundle": bundle_5, "algorithm": "dibbs-enhanced"}
         )
-        assert not response_5.json()["is_match"]
+        person_5 = response_5.json()["person_reference_id"]
+        assert response_5.json()["patient_reference_id"] and uuid.UUID(response_5.json()["patient_reference_id"])
+        assert person_5
+        assert response_5.json()["prediction"] == "no_match"
+        assert not response_5.json()["results"]
 
         bundle_6 = test_bundle
         bundle_6["entry"] = [entry_list[5]]
         response_6 = client.post(
             "/link/fhir", json={"bundle": bundle_6, "algorithm": "dibbs-enhanced"}
         )
-        assert not response_6.json()["is_match"]
+        person_6 = response_6.json()["person_reference_id"]
+        assert response_6.json()["patient_reference_id"] and uuid.UUID(response_6.json()["patient_reference_id"])
+        assert person_6
+        assert response_6.json()["prediction"] == "no_match"
+        assert not response_6.json()["results"]
```

Comment:
I see lots of tests here showing "match" or "no_match", thank you for all of those!  Would it be possible here to also include a test case that highlights a "possible_match"?

Reply:
I test the "possible_match" result (for [Basic](https://github.com/CDCgov/RecordLinker/pull/111/files#diff-42a8bc5fd67e453879d82298e5b7fc88e58a4ecce254e3954af3d87fc0fde518R179-R201) and [Enhanced](https://github.com/CDCgov/RecordLinker/pull/111/files#diff-42a8bc5fd67e453879d82298e5b7fc88e58a4ecce254e3954af3d87fc0fde518R230-R252)) in [test_link.py](https://github.com/CDCgov/RecordLinker/pull/111/files#diff-42a8bc5fd67e453879d82298e5b7fc88e58a4ecce254e3954af3d87fc0fde518). Is that sufficient? Or should we also include tests for this in test_link_router.py?

Reply:
Oh cool, thats the important one!  At this level we'd mostly be testing the if/else logic in the prediction computed field, but that function is pretty simple.  Feel free to skip if its complicated.